### ISSUE TITLE
Brand banners are not synced with foil, kite & ice

### ISSUE THREAD

**Issue by jhdebruin** [djong1/gp3s#6]

**Comment by djong1** [comment]
you can update the banners manually via the moderate menu

**Comment by jhdebruin** [comment]
Let me rephrase it , and make it a feature request :)

"When uploading a new banner to gps-speedsurfing.com , this new banner is replicated automatically to kite, foil and ice websites"

**Comment by jhdebruin** [comment]
Closed

**Comment by jhdebruin** [comment]
Just checking, does this synchronisation also works for spot banners ? I've updated the spot banner for Spot Makkum at gps-speedsurfing.com , but the updated banners doesn't show up at gps-foilsurfing.com 

Example with new banner ->  [ https://www.gps-speedsurfing.com/default.aspx?mnu=forum&forum=2&val=186376 ]
Example with old banner -> [https://www.gps-foilsurfing.com/default.aspx?mnu=user&val=389959&uid=4910&spotid=1762](url)

**Comment by djong1** [comment]
works as designed :-), the idea was to have the ability to run different banners for the same spot based on the source website. So spot banners are not synced across website. Please update the banner also on the foil website manually.

**Comment by jhdebruin** [comment]
Check ✅
